docs/_blog/2025_q2_update.md

```diff
@@ -0,0 +1,56 @@
+- **Language Bindings version 1.2 for iOS, Android, and Python 1.2** The [bdk-ffi 1.2 release](https://github.com/bitcoindevkit/bdk-ffi/releases/tag/v1.2.0) brings Compact Block Filter support through our Kyoto integration, making privacy-preserving light clients accessible to mobile developers. We've also published comprehensive [API documentation](https://bitcoindevkit.org/docs/) for Java and Android bindings.
+
+There are also new projects and initiatives being built.
+- **Silent Payments.** The new [bdk-sp](https://github.com/bitcoindevkit/bdk-sp) crate brings BIP352 Silent Payments functionality to BDK. It has delivered a complete CLI implementation with wallet initialization, PSBT creation and signing, and blockchain scanning capabilities, and full compatibility with BIP352 test vectors and integration examples with bdk-tx.
+- **Transaction Building.** The experimental [bdk-tx](https://github.com/bitcoindevkit/bdk-tx) project represents an evolution of transaction building. This effort to decouple transaction building from the wallet, using rust-miniscript's planning module for optimal satisfaction-weight calculations and the new bdk_coin_select crate for policy-aware coin selection.
+- **Streaming Electrum Client.** The [electrum_streaming_client](https://github.com/bitcoindevkit/electrum_streaming_client) is a streaming, sans-IO Electrum client for asynchronous and blocking Rust applications. This crate provides low-level primitives and high-level clients for communicating with Electrum servers over JSON-RPC. It supports both asynchronous (futures/tokio) and blocking transport models.
+
+## Our Grantees in Action
+
+We're excited to welcome new Silver [corporate members](https://bitcoindevkit.org/blog/_2025q1-new-foundation-members/) the BDK Foundation and thank them for their financial support!
+- AnchorWatch
+- CleanSpark
+- Proton Foundation
+
+We're also excited to welcome new Associate members who are supporting the BDK Foundation by training & funding developers working on BDK!
+- Btrust
+- Vinteum
+
+- ** Grantees**
+BDK Project Maintainer
+- [Leonardo](https://github.com/oleonardolima), funded by Vinteum, continues his work on Tor integration for the Electrum and Esplora crates, enhancing privacy options for BDK users.
+
+Btrust Starter Grantees
+- [Itoro Ukpong](https://github.com/ItoroD), working on the bdk-ffi language bindings and Android example wallet.
+- [Peter Tyonum](https://github.com/tvpeter), upgrading bdk-cli to the latest bdk rust libraries and adding new features.
+
+
+Thunderbiscuit is taking a 2-month break from BDK using an HRF grant to create an iOS version of [Padawan Wallet](https://play.google.com/store/apps/details?id=com.coyotebitcoin.padawanwallet)
```

Comment:
@thunderbiscuit any recommendation on how I should write this sentence better than what I did here?

Reply:
No it's good!